wire-ios/Wire-iOS/Sources/UserInterface/AddContacts/AddParticipantsViewController.swift

```diff
@@ -179,7 +179,8 @@ final class AddParticipantsViewController: UIViewController, SpinnerCapable {
                                                                   shouldIncludeGuests: viewModel.context.includeGuests,
                                                                   isFederationEnabled: isFederationEnabled)
 
-        emptyResultView = EmptySearchResultsView(isSelfUserAdmin: SelfUser.current.canManageTeam,
+        let user = SelfUser.provider?.providedSelfUser
+        emptyResultView = EmptySearchResultsView(isSelfUserAdmin: user?.canManageTeam == true,
```

Comment:
There are some places likes this one where you don't use `assertionFailure`, why did you decide to skip it here for example?

Reply:
Because I think a crash/`assertionFailure` is not really needed to debug here. Basically it just checks if `isSelfUserAdmin` and in case there is no user, the boolean is `false` (which seems to be logical and correct).